Sources/Hummingbird/Server/Request.swift

```diff
@@ -50,23 +50,9 @@ public struct HBRequest: Sendable, HBSendableExtensible {
     public var headers: HTTPHeaders { self._internal.headers }
     /// Body of HTTP request
     public var body: HBRequestBody
-    /// Logger to use
-    public var logger: Logger
-    /// reference to application
-    public var applicationContext: HBApplication.Context { self._internal.applicationContext }
     /// Request extensions
     public var extensions: HBSendableExtensions<HBRequest>
-    /// Request context (eventLoop, bytebuffer allocator and remote address)
-    public var context: HBRequestContext { self._internal.context }
-    /// EventLoop request is running on
-    public var eventLoop: EventLoop { self._internal.context.eventLoop }
-    /// ByteBuffer allocator used by request
-    public var allocator: ByteBufferAllocator { self._internal.context.allocator }
-    /// IP request came from
-    public var remoteAddress: SocketAddress? { self._internal.context.remoteAddress }
-
-    /// Request ID (Uses ID attached to loggger to avoid replication)
-    public var id: String { self.logger[metadataKey: "hb_id"]!.description }
+    public let id: String
```

Comment:
Is this something we can move to context as well? I was kind of hoping we could ditch `HBRequest` for `HBHTTPRequest`. The ID was something you asked for initially what was the intention for it.

Reply:
In fact the id is stored in both request and context at the moment